demo/src/faber.ts

```diff
@@ -0,0 +1,155 @@
+import type { ConnectionRecord } from '@aries-framework/core'
+import type { CredDef, Schema } from 'indy-sdk-react-native'
+import type BottomBar from 'inquirer/lib/ui/bottom-bar'
+
+import { CredentialPreview, ProofAttributeInfo, AttributeFilter } from '@aries-framework/core'
+import { uuid } from '@aries-framework/core/build/utils/uuid'
```

Comment:
This still contains the build path, won't run for me like this

```suggestion
import { utils } from '@aries-framework/core'

utils.uuid()
```